src/temporal/t.rast.boxplot/t.rast.boxplot.py

```diff
@@ -1,631 +1,802 @@
+        if vertical:
+            ax.xaxis.set_major_formatter(date_fmt)
+            ax.set_xlim(mdates.num2date(start_time), mdates.num2date(end_time))
+        else:
+            ax.yaxis.set_major_formatter(date_fmt)
+            ax.set_ylim(mdates.num2date(start_time), mdates.num2date(end_time))
+    if "minute" in temp_unit:
+        if bool(date_format):
+            date_fmt = mdates.DateFormatter(date_format)
+        else:
+            date_fmt = mdates.DateFormatter("min. %M")
+        start_time = mdates.date2num(rast_dates[0]) - 1 / 2880 * temp_lngt
+        end_time = mdates.date2num(rast_dates[-1]) + 1 / 2880 * temp_lngt
+        if vertical:
+            ax.xaxis.set_major_formatter(date_fmt)
+            ax.set_xlim(mdates.num2date(start_time), mdates.num2date(end_time))
+        else:
+            ax.yaxis.set_major_formatter(date_fmt)
+            ax.set_ylim(mdates.num2date(start_time), mdates.num2date(end_time))
+    if "second" in temp_unit:
+        if bool(date_format):
+            date_fmt = mdates.DateFormatter(date_format)
+        else:
+            date_fmt = mdates.DateFormatter("sec. %S")
+        start_time = mdates.date2num(rast_dates[0]) - 1 / 172800 * temp_lngt
+        end_time = mdates.date2num(rast_dates[-1]) + 1 / 172800 * temp_lngt
+        if vertical:
+            ax.xaxis.set_major_formatter(date_fmt)
+            ax.set_xlim(mdates.num2date(start_time), mdates.num2date(end_time))
+        else:
+            ax.yaxis.set_major_formatter(date_fmt)
+            ax.set_ylim(mdates.num2date(start_time), mdates.num2date(end_time))
+    return ax
+
+
+def main(options, flags):
+    """
+    Draws the boxplot of raster values. Optionally, this is done per
+    category of a zonal raster layer
+    """
+
+    global matplotlib
+    global plt
+
+    # lazy import matplotlib
+    try:
+        import matplotlib
+
+        matplotlib.use("WXAgg")
+        from matplotlib import pyplot as plt
+    except ModuleNotFoundError:
+        gs.fatal(_("matplotlib is not installed"))
+
+    # Plot format options
+    fontsize = options["fontsize"]
+    if fontsize:
+        int(fontsize)
+        plt.rcParams["font.size"] = fontsize
+    else:
+        fontsize = 10
+    if options["rotate_labels"]:
+        rotate_label = float(options["rotate_labels"])
+    grid = flags["g"]
+
+    # Get range (if defined)
+    whisker_range = get_range(options["range"])
+
+    # Get flier size, marker and color
+    flier_size, flier_marker, flier_color = get_flier_options(
+        options["flier_size"], options["flier_marker"], options["flier_color"]
+    )
+
+    # Output options
+    output = options["output"]
+    dpi, dimensions, vertical = get_output_options(
+        options["dpi"], options["plot_dimensions"], flags["h"]
+    )
+
+    # Get names, dates, tic position and temporal type of raster layers
+    strds = options["input"]
+    rast_names, rast_dates, tic_positions, temp_type = get_rast_name_dates(strds)
```

Comment:
Here it would better use `gs.parse_command()` func with `parse` parameter.

```python
    # Get names, dates, tic position and temporal type of raster layers
    rast_names, rast_dates, tic_positions, temp_type = gs.parse_command(
        "t.rast.list",
        flags="u",
        input=options["input"],
        columns=["name", "start_time", "temporal_type"],
        parse=(get_rast_name_dates, {"col_sep": "|"}),
    )
```

Documentation for `gs.parse_command()` func is [here](https://grass.osgeo.org/grass80/manuals/libpython/script.html?highlight=parse_command#script.core.parse_command).